contribs/gmf/src/controllers/vars_desktop.scss

```diff
@@ -51,4 +52,7 @@ $notification-width: 12.5rem !default;
   --right-panel-width-googlestreetview: 25rem;
   --right-panel-width-mapillary: 25rem;
   --left-panel-width: #{$left-panel-width};
+  --panel-separator-width: #{$panel-separator-width};
+  --current-left-panel-separator-width: #{$panel-separator-width};
+  --left-panel-border-width: #{$border};
```

Comment:
I think that good if you rename it to `--current-left-panel-border-width` then we know that it can't be set by the config...